### DISCUSSION TITLE
Social Platform icons

### DISCUSSION THREAD

**Discussion by marcofranssen** [tailwindlabs/heroicons#491] [Ideas]
It would  be great to have social platform icons available that can be used to link to social platforms. E.g. when building a blog or company webpage it would be great to create some nice stylish links to the companies or personal social profiles.

- [ ] Facebook
- [ ] Twitter
- [ ] LinkedIn
- [ ] Keybase
- [ ] Instagram
- [ ] Github
- [ ] Youtube
- [ ] StackOverflow
- [ ] Reddit
- [ ] 500px
- [ ] Flickr
- [ ] Unsplash
- [ ] ...

That would prevent me from including font-awesome so I can stick completely with the tailwind stack.

**Comment by dingman** [comment]
+1

**Comment by chrissuperduper** [comment]
+1

**Comment by gidea** [comment]
+1 as well ;)

**Comment by Nefcanto** [comment]
So, what's the status of this thread? Does Tailwind team have any plan to support this? I know we can use SVGs directly. But there are two major drawbacks to working with SVGs:

- First you lose consistency and you should be hugely talented to choose correct SVGs from different sites to match them up
- Second you lose code readability and ease of use. SVG is very verbose and dirty.

**Comment by saddit** [comment]
Replacing Font-awesome is a daydream in a short time 😄

**Comment by huv1k** [comment]
Any news regarding this? 🤔

**Comment by thepixelmonk** [comment]
Any news from tailwind on this? Seems like a pretty obvious addition to the icon set.

**Comment by divineslight** [comment]
Can the authors/maintainers provide their point of view rather than silently closing all discussions surrounding this request?

**Comment by huksley** [comment]
So TailwindUI recommends this icon set, before buying one goes to check is the GitHub icon or other logo icons are present... and there is none? Even if it is used in demo templates?

Please clarify the source of logo icons in templates, for example in https://spotlight.tailwindui.com/

**Reply by thepixelmonk** [reply]
> So TailwindUI recommends this icon set, before buying one goes to check is the GitHub icon or other logo icons are present... and there is none? Even if it is used in demo templates?
> 
> Please clarify the source of logo icons in templates, for example in https://spotlight.tailwindui.com/

Yeah exactly the fact random raw social svgs are all over the place in their advertising and templates and then not having them in the actual icon set is pretty weird.

**Comment by lux10n** [comment]
+1
yeah, i need these

**Comment by marcofranssen** [comment]
Seems there is a lot of desire for this, but after 2 years no initiatives to actually do this? What is required to get some things moving here?

**Comment by zefir-git** [comment]
You could use https://simpleicons.org — Simple SVG icons of thousands of brands (free and open-source https://github.com/simple-icons/simple-icons). The website also tells you what their primary brand colour is and has a link to logo usage guidelines if present.

I personally don‘t think heroicons needs brand icons since these are usually already provided by the brand as SVG and there isn‘t much you can adapt them to make them fit the rest of the heroicons, i.e. you can‘t edit a company‘s logo just so that it matches your website (at least afaik). And when the brand doesn‘t provide an SVG on their "Brand usage" or similar page, you can often find it online. One good place (if you don‘t want to use simpleicons.org) is Wikipedia. It often has an SVG logo of the brand when you open its page (sometimes it's a PNG rendering and there's a link labelled something like "Show original SVG"). And many other websites also provide brand logos.

> **Note**: To use simpleicons.org with text colour (i.e. `text-blue-500` in TW, etc.) you need to add `fill="currentColor"` to the `<svg>` tag.

**Comment by ActaV-N** [comment]
+1
really good idea
